eos/parameters/hadronic-matrix-elements.yaml

```diff
@@ -1875,692 +2184,828 @@ groups:
           central: 0.7
           min:     0.7
           max:     0.7
+          latex:    '$s_{0_{{+,0}} }^{B \to \pi, \text{(B-LCSR)}}$'
```

Comment:
Why do you have double brackets?

Reply:
This $0_{+,0}$ looks a bit weird, no?

Reply:
"Why do you have double brackets?": cause my regex failed at some points "{" and "}" are weird characters to use in the built-in substitute function of vim... Is that different in emacs? :P
"This $0_{+,0}$ looks a bit weird, no?": hmm... Not sure. I think it looks fine ...

Reply:
There is no such things as "problematic" in emacs :)